README.md

```diff
@@ -154,6 +154,8 @@ These mechanisms work great in most cases. However, they do have some limitation
 
 * It is possible that a module does its own queuing of callback functions – effectively merging asynchronous execution contexts. For example, one may write an http request buffering library that queues requests and then performs them in a batch in one shot. In such a case, when all the callbacks fire, they will execute in the context which flushed the queue instead of the context which added the callbacks to the queue. This problem is called the pooling problem or the [user-space queuing problem][queuing-problem], and is a fundamental limitation of JavaScript. If your application uses such code, you will notice that RPCs from many requests are showing up under a single trace, or that certain portions of your outbound RPCs do not get traced. In such cases we try to work around the problem through monkey patching, or by working with the library authors to fix the code to properly propagate context. However, finding problematic code is not always trivial.
 * If your application uses untranspiled `async` functions, you must use Node 8+. (Untranspiled `async` functions are supported from Node 7.6 onward, but we do not support tracing these functions in Node 7.)
+* Requests originating from the body of a `then` implementation in a userspace thenable may not get traced if the thenable is `await`-ed. This is largely an unconventional case but is present in the `knex` module, which monkeypatches the Bluebird Promise's prototype to make database calls. __If you are using `knex` (esp. the `raw` function), see [https://github.com/googleapis/cloud-trace-nodejs/issues/946](#946) for more details on whether you are affected, as well as a suggested workaround.__
```

Comment:
`userspace thenable` may not be well defined enough to most casual readers. How about something like this:

* Node.js `async_hooks` API has issues tracking context around `await`-ed "thenables" (rather than real promises). Requests originating from the body of a `then` implementation in such a user-space "thenable" may not get traced. This is largely an unconventional case but is present in the `knex` module, which monkeypatches the Bluebird Promise's prototype to make database calls. __If you are using `knex` (esp. the `raw` function), see [https://github.com/googleapis/cloud-trace-nodejs/issues/946](#946) for more details on whether you are affected, as well as a suggested workaround.__

Reply:
Thanks, I changed it